template-stand/docker-compose.yaml

```diff
@@ -0,0 +1,54 @@
+clickhouse:
+  image: yandex/clickhouse-server
+  ports:
+  - "8124:8123"
+  volumes:
+  - "./rollup.xml:/etc/clickhouse-server/config.d/rollup.xml"
+  - "./data/clickhouse/data:/var/lib/clickhouse/data"
+  - "./data/clickhouse/metadata:/var/lib/clickhouse/metadata"
+clickhouse-init:
+  image: yandex/clickhouse-server
+  volumes:
+  - "./init.sql:/init.sql"
+  entrypoint: |
+    bash -c '
+      sleep 5
+      export HOME=/var/lib/clickhouse/
+      cat /init.sql | clickhouse client --host clickhouse --multiquery'
+  links:
+  - clickhouse
+carbon-clickhouse:
+  image: sham42/carbon-clickhouse:latest
```

Comment:
could you please provide a link here?

Reply:
https://github.com/mary3000/carbon-clickhouse